AnnoyancesFilter/sections/popups.txt

```diff
@@ -465,7 +465,7 @@ xgamers.ru###SubscribeDV
 xgamers.ru###BackFon_DataBlock
 petit-mag.com###sticknav
 braze.com###sticky-toaster
-sk7mobile.com##.modal-backdrop
+diziweb.net,tfc.tv,bkv.hu,catwalkyourself.com,baba-mail.co.il,egyptianstreets.com,ports.com.ua,bitfeed.co,calkalasutasayrani.com,sk7mobile.com,blogkurdu.net,opensnow.com,watch5s.rs##.modal-backdrop
```

Comment:
This one we've investigated with @wout-codes here https://github.com/AdguardTeam/AdguardFilters/pull/76412. Please remove this changes, but you can help find obsolete domains that don't need this rule.